### PR TITLE
taskwarrior-tui: init at 0.9.5

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

`This PR was brought to you by someone who has no push rights to nixos/nixpkgs, and thus you can trust that this PR did not delete anything in nixos/nixpkgs when it was created.`

### CODE REVIEW

File: pkgs/applications/misc/taskwarrior-tui/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  version = "0.9.5";
+
+  src = fetchFromGitHub {
+    owner = "kdheepak";
+    repo = "taskwarrior-tui";
+    rev = "v${version}";
+    sha256 = "1348ypjphm5f46civbrcxbbahwwl2j47z1hg8ndq1cg2bh5wb8kg";
+  };
+
+  # Because there's a test that requires terminal access
```
Comment (symphorien) [inline pkgs/applications/misc/taskwarrior-tui/default.nix:17]: if this is only one test among many, please disable it with a patch or sed.

**Comment by symphorien** [comment]
> This PR was brought to you by someone who has no push rights to nixos/nixpkgs, and thus you can trust that this PR did not delete anything in nixos/nixpkgs when it was created.

I think this was unnecessary.

File: pkgs/applications/misc/taskwarrior-tui/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  doCheck = false;
+
+  cargoSha256 = "11zpy3whzir9mlbvf0jyscqwj9z44a6s5i1bc2cnxyciqy9b57md";
+
+  meta = with stdenv.lib; {
+    description = "A terminal user interface for taskwarrior ";
+    homepage = "https://github.com/kdheepak/taskwarrior-tui";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ matthiasbeyer ];
```
Comment (symphorien) [inline pkgs/applications/misc/taskwarrior-tui/default.nix:26]: platforms is missing

File: pkgs/applications/misc/taskwarrior-tui/default.nix (hunk 1 above)
Comment (matthiasbeyer) [reply pkgs/applications/misc/taskwarrior-tui/default.nix:17]: One among four afaics, I think value of these three other tests vs. effort to disable the one is too much with too little benefit.

Comment (symphorien) [reply pkgs/applications/misc/taskwarrior-tui/default.nix:17]: `substituteInPlace src/main.rs --replace 'fn test_main' $'#[ignore]\nfn test_main'`

File: pkgs/applications/misc/taskwarrior-tui/default.nix (hunk 2 above)
Comment (SuperSandro2000) [reply pkgs/applications/misc/taskwarrior-tui/default.nix:26]: Platforms for buildRustPackage should only be set if it is not all or unix. Please remove that again.

File: pkgs/applications/misc/taskwarrior-tui/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/misc/taskwarrior-tui/default.nix:17]: I think it is fine here to disable tests.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108592` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>taskwarrior-tui</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108592` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>taskwarrior-tui</li>
</ul>
</details>

**Pull request merged**